src/config/field-overrides/program-indicator/OrgUnitField.js

```diff
@@ -175,7 +203,7 @@ export default class OrgUnitField extends Component {
         return false;
     }
 
-    shouldFetchDataElementsForProgram() {
+    requiresDataElementsForProgram() {
```

Comment:
The comments in this function doesn't really match the name of the function anymore. I actually think the previous name was a bit clearer?
You could just keep the `requiresDataElementsForProgram` as name of the variable of the result of this.